src/main/java/duke/ui/UI.java

```diff
@@ -0,0 +1,55 @@
+package duke.ui;
+
+import duke.task.TaskList;
+
+public class UI {
+
+    public void greet() {
+        String greeting = "____________________________________________________________\n"
+                + "Hello! I'm Duke\n"
+                + "What can I do for you?\n"
+                + "____________________________________________________________\n";
+        System.out.println(greeting);
+    }
+
+    public void farewell() {
+        String farewell = "Bye. Hope to see you again soon!\n"
+                + "____________________________________________________________\n";
+        System.out.println(farewell);
+    }
+
+    public void markedAsDone(int index, TaskList tasks) {
```

Comment:
markedAsDone is quite similar with TaskList Class' markAsDone, should we change to printDoneNotification or printDoneResult?

Reply:
Indeed this causes ambiguity, including my other print-related methods. Thanks!